diff --git a/dist/esm/worker.js b/dist/esm/worker.js
index cca2a6116bab5349cfa35bc1dba0b6e696af91ee..17cbffe06475c002bdb0f57b6f77e7a364d294bb 100644
--- a/dist/esm/worker.js
+++ b/dist/esm/worker.js
@@ -17,7 +17,7 @@ const load = async ({ coreURL: _coreURL, wasmURL: _wasmURL, workerURL: _workerUR
             _coreURL = CORE_URL.replace('/umd/', '/esm/');
         // when web worker type is `module`.
         self.createFFmpegCore = (await import(
-        /* @vite-ignore */ _coreURL)).default;
+        /* webpackIgnore: true */ /* @vite-ignore */ _coreURL)).default;
         if (!self.createFFmpegCore) {
             throw ERROR_IMPORT_FAILURE;
         }
